src/main/java/james/task/Task.java

```diff
@@ -0,0 +1,47 @@
+package james.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
boolean naming follows best practices 👍🏻

Reply:
thank you